internal/cli/aux_test.go

```diff
@@ -0,0 +1,14 @@
+package cli
+
+import (
+	"log"
+	"os"
+	"testing"
+)
+
+func TestSetHelmChartsPathVar(t *testing.T) {
+	setHelmChartsPathVar("/test")
+	if os.Getenv("CHARTS_PATH") != "/test" {
```

Comment:
You should use the Go Assertion util https://github.com/stretchr/testify that is used in other tests `assert.Equal(t, "/test", os.Getenv("CHARTS_PATH"))